lib/watchbuild/runner.rb

```diff
@@ -86,6 +94,32 @@ def app
       @app ||= Spaceship::Application.find(WatchBuild.config[:app_identifier])
     end
 
+    def notify_slack(message, url)
+      require 'net/http'
+      require 'uri'
+      require 'json'
+
+      slack_url = uri = URI.parse(url)
+      slack_message = {
+        "text": message
+      }
+
+      header = {'Content-Type': 'application/json'}
+  
+      http = Net::HTTP.new(slack_url.host, slack_url.port)
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
This code is pretty dangerous as it means we won't do any form of validation of the upstream server, which can be trivially mitmd. Is there a reason why you're disabling certificate verificatoin?

Reply:
Had disabled during development while debugging a network issue, will fix up - thanks 👍